### ISSUE TITLE
'Undefined namespace prefix' errors with 'skip' css selector after upgrading to 0.1.0

### ISSUE THREAD

**Issue by reimund** [rc0x03/node-osmosis#63]
Maybe I'm missing something, but I can't get :skip(n) to work after upgrading to 0.1.0.

Did usage change?

Getting a lot of these:

```
xmlXPathEval: evaluation failed
XPath error : Undefined namespace prefix
.//dd:skip(1)
          ^
xmlXPathEval: evaluation failed
XPath error : Undefined namespace prefix
.//dd:skip(2)
````

I'm simply doing:
```
.set({
	foo:    'dd:first',
	bar:    'dd:skip(1)',
	foobar: 'dd:skip(2)',
})
```

**Comment by rc0x03** [comment]
This is a known issue in the latest version on libxmljs-dom. The next version has much better css2xpath support. Once I finish creating tests for all the selectors, I'll upload the latest version.

**Comment by rc0x03** [comment]
Fixed in 0.1.1
